content/50.discussion.md

```diff
@@ -9,6 +9,69 @@ In this review, we seek to explain the scientific rationale underlying these tec
 
 ### Current State of Therapeutics
 
+The majority of current clinical trials and lines of investigation have focused on repurposing existing therapies to counter SARS-CoV-2 and treat its symptoms.
+This approach is necessary given the urgency of the situation as well as the extensive time required for developing and testing new therapies.
+However, in the long-term, new drugs specific for treatment of COVID-19 may also enter development.
+There is thus value in investigating two lines of inquiry for treatment of COVID-19: 1) new therapeutics specific for treatment of COVID-19 or its symptoms, and 2) repurposing of existing therapeutics for treatment of COVID-19 or its symptoms.
+Here we consider further avenues that scientific investigators may explore in the development of therapies for COVID-19.
+
+Given the great focus in investigating hydroxychloroquine (HCQ) as a potential antiviral treatment for SARS-CoV-2, it may be of interest to researchers to explore related alternatives.
```

Comment:
Does this need to be rephrased as there is less interest now due to lack of efficacy, maybe saying " looking to alternatives" instead

Reply:
```suggestion
While studies of hydroxychloroquine (HCQ) have not supported the early theoretical interest in this antiviral treatment, it may be of interest to explore alternatives with similar mechanisms.
```
I haven't edited this section yet but good idea to tone down the HCQ naiveté wherever we find it! haha